desktop/conf.dist/hue.ini

```diff
@@ -1277,6 +1277,18 @@
       # AWS region to use
       ## region=us-east-1
 
+      # Endpoint overrides
```

Comment:
Could you add the same in https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl too? (those are almost the same ini files to keep in sync)